go-controller/pkg/config/config.go

```diff
@@ -91,6 +91,9 @@ var (
 
 	// EnableMulticast enables multicast support between the pods within the same namespace
 	EnableMulticast bool
+
+	// IPv6Mode captures whether we are using IPv6 for OVN logical topology
+	IPv6Mode bool
```

Comment:
Maybe `IPv6SingleStack`? (We're going to start working on dual-stack support soon.)
otherwise lgtm

Reply:
@danwinship @russellb I assume that dual-stack will basically replace single-stack? Not sure why we'd keep both since dual-stack is mostly a superset of single.

Reply:
I would expect to eventually support any of: v4-only, v6-only, v4+v6

I don't know how this would look in the config.  I wouldn't worry about it too much right now.  We're going to refactor a bunch no matter what to make v4+v6 work ...

Reply:
@danwinship is the `mode` in IPv6Mode confusing? I think, like @russellb says, when we move to the dual-stack mode there will be some changes around this configuration variable (and perhaps we might need additional variables). Let me know if you strongly feel it to be changed to IPv6SingleStack, I will change it. Thanks.